### PR TITLE
fix(textfield): add primary color to textfield label on focus

### PR DESCRIPTION
**Pull request by williamernest**
fixes: #1435

### CODE REVIEW

**Comment by williamernest** [comment]
@amsheehan 
Changed it to apply the label color from the input focus psuedo class.

File: packages/mdc-textfield/mdc-text-field.scss [hunk 1]
```diff
@@ -237,7 +241,6 @@
     position: absolute;
     bottom: 20px;
     width: calc(100% - #{$mdc-text-field-icon-padding});
-    color: $mdc-text-field-box-secondary-text;
```
Comment (acdvorak) [inline packages/mdc-textfield/mdc-text-field.scss]: Why does this line need to be removed? It doesn't appear to be related to the `:focused` state.

Also, it looks like `$mdc-text-field-box-secondary-text` isn't used anywhere else, so if we really do need to remove this line, we should also delete the variable from `_variables.scss`.

File: packages/mdc-textfield/mdc-text-field.scss [hunk 2]
```diff
@@ -246,10 +249,6 @@
     // when the ripple is activated behind it.
     will-change: transform;
 
-    @include mdc-theme-dark(".mdc-text-field") {
```
Comment (acdvorak) [inline packages/mdc-textfield/mdc-text-field.scss]: Why do these lines need to be removed? They don't appear to be related to the `:focused` state.

File: packages/mdc-textfield/mdc-text-field.scss (hunk 1 above)
Comment (williamernest) [reply packages/mdc-textfield/mdc-text-field.scss]: The default `color` property is set in the mdc-text-field-label.scss file. My idea was to let the label scss file determine the default values, and then override them when we need to change them based on interaction with the other elements in the mdc-text-field.scss file.
